### DISCUSSION TITLE
JNI DETECTED ERROR IN APPLICATION

### DISCUSSION THREAD

**Discussion by jronald-wowfingers** [software-mansion/react-native-screens#2508] [Q&A]
Hi , 

I am getting the error :
"JNI DETECTED ERROR IN APPLICATION: JNI GetObjectRefType called with pending exception java.lang.UnsatisfiedLinkError: No implementation found for com.facebook.jni.HybridData com.swmansion.rnscreens.NativeProxy.initHybrid() (tried Java_com_swmansion_rnscreens_NativeProxy_initHybrid and Java_com_swmansion_rnscreens_NativeProxy_initHybrid__)"

 when  I try to run the build on an emulator, it opens the app and closes almost immediately,

I got the following error from looking at the emulator logs.

I am running the following dependancies: 
                "@react-navigation/drawer": "^7.0.3",
		"@react-navigation/native": "^7.0.2",
		"@react-navigation/native-stack": "^7.0.2",
		"react-native-safe-area-context": "^4.14.0",
		"react-native-screens": "^4.1.0"

Can you please help with regards to what this error means and where I should look to fix it or whether I should log it as a bug ?

**Comment by kkafar** [comment]
@jronald-wowfingers THe error message indicates that the `librnscreens.so` has not been loaded by JVM. This could mean that the library has not built properly. I think you should start with cleaning up all the caches in your project.

`rm -fr node_modules android/build android/app/build android/app/.cxx && yarn install` and then try out fresh build.

**Comment by kkafar** [comment]
I'll close the discussion as inactive. If the issue persists or anyone encounters something similar please file a regular issue ticket.
